src/lens/article/nodes/related_article/related_article_view.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+var NodeView = require("../node").View;
+var $$ = require("../../../substance/application").$$;
+
+
+// Substance.RelatedArticle.View
+// ==========================================================================
+
+var RelatedArticleView = function(node, viewFactory) {
+  NodeView.call(this, node, viewFactory);
+};
+
+RelatedArticleView.Prototype = function() {
+  this.render = function() {
+    NodeView.prototype.render.call(this);
+
+    const $content = $$('.related-articles');
+
+    const articles = this.node.properties.articles;
```

Comment:
not to append empty $content - maybe check if articles is empty  first, and return if empty at once (before line 18) ?